test/lib.bash

```diff
@@ -427,35 +427,32 @@ function run_rolling_upgrade_tests {
 
   logger.info "Running rolling upgrade tests"
 
-  local base serving_image_version eventing_image_version eventing_kafka_broker_image_version image_template common_opts images_file
+  local image_template common_opts images_file
 
   # Specify image mapping for REKT tests
   images_file="$(dirname "$(realpath "${BASH_SOURCE[0]}")")/images-rekt.yaml"
-  serving_image_version=${KNATIVE_SERVING_VERSION#knative-}
-  eventing_image_version="${KNATIVE_EVENTING_VERSION#knative-}"
-  eventing_kafka_broker_image_version="${KNATIVE_EVENTING_KAFKA_BROKER_VERSION#knative-}"
+
+  default_knative_eventing_images
+  default_knative_eventing_kafka_broker_images
+  default_knative_serving_images
 
   # Mapping based on https://github.com/openshift/release/tree/master/core-services/image-mirroring/knative
   # for non-REKT tests.
-  base="quay.io/openshift-knative/"
```

Comment:
The images you're replacing here must be multi-arch as they're used on P/Z now.
See e.g. https://quay.io/repository/openshift-knative/eventing/eventshub?tab=tags&tag=latest - it's multiarch.

Reply:
@rudyredhat1 what's the status of multi arch builds ?

Reply:
sorry missed this notification, checked now, I think it will take time, trying to understand how the kustomize patches can be used for the docket-build-oci-ta.yaml , i will check this weekend and will try to update something on this MR - https://github.com/openshift-knative/hack/pull/316 , cc @pierDipi